web/src/lib/components/forms/library-import-path-form.svelte

```diff
@@ -5,15 +5,27 @@
   import FullScreenModal from '../shared-components/full-screen-modal.svelte';
   import { mdiFolderSync } from '@mdi/js';
 
-  export let importPath: string;
+  import { onMount } from 'svelte';
```

Comment:
can you not put new lines between import statements please?

Reply:
Oh, my bad. Will fix